internal/pkg/mapper/sharedcode/codes/save_test.go

```diff
@@ -22,39 +22,15 @@ func TestSharedCodeSaveMissingKey(t *testing.T) {
 	assert.Len(t, recipe.ExtraFiles, 0)
 }
 
-func TestSharedCodeSaveString(t *testing.T) {
```

Comment:
`TestSharedCodeSaveString` test zmazany, riesi sa to pri load ... tj. interna reprezentacia uz nemoze byt `string`.